app/views/judgment/judgmentHomepage.scala.html

```diff
@@ -25,13 +25,27 @@ <h1 class="govuk-heading-xl">@title</h1>
           }
         }
 
-        <h2 class="govuk-heading-m">If this is an update to an existing judgment or decision</h2>
-        <p class="govuk-body">You can use this service to transfer an update or revision to a previously transferred document.</p>
-        <p class="govuk-body">Transfer the document in the same way as any judgment or decision, by clicking "Start your transfer" above.</p>
-        <p class="govuk-body">Once you have successfully completed the transfer you will need to email us. More information will be provided after the transfer.</p>
-        <h2 class="govuk-heading-m">Contact the publishing editors</h2>
-        <p class="govuk-body"><a href="mailto:@Messages("nationalArchives.judgmentsEmail")">@Messages("nationalArchives.judgmentsEmail")</a></p>
-
+        @if(!blockPressSummaries) {
+            <h2 class="govuk-heading-m">Service update – September 2025</h2>
```

Comment:
There is a small change in the content. Shakir has added a comment on the ticket. https://national-archives.atlassian.net/browse/TDRD-996?focusedCommentId=108821